nicolegooden/indecision-maker
# Initial react setup bd

#### What's this PR do?  

- Adds folders for all components with test / css / component file nested 
- Adds dependancies for various packages needed for testing/routing/react
- Adds `apiCalls` for all activities and genres if applicable 
- Adds initial structure for APP and homepage

#### Where should the reviewer start? 

- Start by running `npm install` to install necessary dependencies 

#### How should this be manually tested?  

- Run `npm start` to load the application in the browser 
- Check for file inconsistencies throughout the application

#### Any background context you want to provide?  

- Still debating how the data / action flow should work for our application  .

#### What are the relevant tickets?  

closes #32
